symengine/pow.h

```diff
@@ -34,12 +34,12 @@ class Pow : public Basic
     //! \return `true` if canonical
     bool is_canonical(const Basic &base, const Basic &exp) const;
     //! \return `base` of `base**exp`
-    inline RCP<const Basic> get_base() const
+    inline const RCP<const Basic>& get_base() const
     {
         return base_;
     }
     //! \return `exp` of `base**exp`
-    inline RCP<const Basic> get_exp() const
+    inline const RCP<const Basic>& get_exp() const
```

Comment:
It is safe to return a raw reference as long as you use it on the same line, like `get_exp().diff(x)`.